### DISCUSSION TITLE
EME/WP Repository/Github...

### DISCUSSION THREAD

**Discussion by TRILOS** [liedekef/events-made-easy#29] [General]
Hi,
I´ve just noticed the security issue by itsec-site-scanner.ithemes.com (and wondering about 3 weeks delay) and read https://www.e-dynamics.be/wordpress/forums/topic/security-release/
I think it´s not clear yet what the plan for EME is, whether it will be back in WP repo, my way to handle EME on a customer´s site (not to forget translations) depends on this and I kindly ask for more information on the future of this plugin.
BG Erik

**Comment by liedekef** [comment]
I will continue to update EME as usual, but I gave up in trying to be compliant to the WP coding standards. I could give many examples as to why they are ridiculous and no actual plugin is compliant. I just got the bad luck to have a minor security issue and now they would require me to work for weeks/months and even disable some things because no longer allowed ...
The translations are still handled too, see the readme for that. I am still searching for something more automatic, but for now they are packaged in the code.

**Reply by liedekef** [reply]
Btw: once you install EME from the github releases, it will update as usual (automatically and/or in your wp admin dashboard). And it will stay free as always.
